signer/threshold_signer_hsm.go

```diff
@@ -0,0 +1,44 @@
+package signer
```

Comment:
I don't think it's necessary to include a file introducing an unusable type.

Reply:
The reasons.
1. To have a blueprint for future implementation. 
2. As a cliff-hanger for the future
3. For "maintainability"  and code overview.